web-api/src/storage/elastic.rs

```diff
@@ -334,7 +339,7 @@ impl Client {
                     .hits
                     .hits
                     .into_iter()
-                    .map(|hit| (hit.id, hit.score))
+                    .map(|hit| (hit.id, hit.score / multi_search_correction_factor))
```

Comment:
:question: How can this bring the result from ES back to `(0..1)`? I think we need to rescale them. We should also consider that if the score range from BM25 is higher than knn query, then, we are only scoring based on that.

Reply:
I don't see much value in applying this. Having 5 as a score instead of 10 will not make it sensibly better and we are confusing the code.
I'll prefer to tackle this properly in the future.